modules/tictactoe/tictactoe.js

```diff
@@ -148,7 +148,7 @@ function checkWin(game, player) {
         if (row[0] === piece && row[1] === piece && row[2] === piece) {
             return player;
         }
-    }   
+    }
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected space or tab after "//" in comment.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529962506/issues/source?bid=3282987&fileBranchId=3282998#l153)